server/graphql/v2/object/User.js

```diff
@@ -0,0 +1,130 @@
+import { GraphQLBoolean, GraphQLInt, GraphQLList, GraphQLObjectType, GraphQLString } from 'graphql';
+import { GraphQLDateTime } from 'graphql-iso-date';
+
+import models, { Op } from '../../../models';
+
+import { Collective } from './Collective';
+import { Member } from './Member';
+
+export const User = new GraphQLObjectType({
+  name: 'UserDetails',
+  description: 'This represents the details of a User',
+  fields: () => {
```

Comment:
Please don't introduce any field not required by the current work. Or the review process is going out of track.

Reply:
@znarf : Ah, I somehow initially thought we need the whole object, but I don't think we do; we only need the fields we try to query from the frontend. Thus I have adjusted this. 😄 Hope now it looks cleaner. 😄